pymc3/distributions/__init__.py

```diff
@@ -81,11 +81,7 @@
 from .timeseries import MvGaussianRandomWalk
 from .timeseries import MvStudentTRandomWalk
 
-from .transforms import transform
-from .transforms import stick_breaking
-from .transforms import logodds
-from .transforms import log
-from .transforms import sum_to_1
+from .transforms import *
```

Comment:
I like this better than the current method. Two things:

1. This is technically a breaking change, since `pm.distributions.transform` will no longer work. I bet this messes up 0 people, but we should add a deprecation warning before removing.
2. As a suggestion, I think we should surface transforms as `pm.transforms`, so that you could use `pm.transforms.ordered()` instead of `pm.distributions.ordered()` (In fact, I think right now `pm.log` might be a transform, and not a logarithm?). I _think_ we would do this by not importing anything from `.transforms` here, but in the top level init, `import .distributions.transforms as transforms`. How does that sound?

Reply:
WRT your (1), I think we could just not export `transform`, since it's just an alias for `Transform`.  That's also theoretically a breaking change, but seems even less likely to cause troubles.  Especially since one should not use `transform` as the name of a class in python.

(2) Sounds good to me.

LMK what you think about my responses and I will make the changes in the PR, or I believe you could do them yourself, if that's what you prefer.

Reply:
Both of those sound good to me. Feel free to ping after you mess with those imports I'll kick the tires locally (and add tests if I find anything :) ) and then merge. Ooh, I'll check the doc thing you just mentioned, too.

Reply:
Wait -- why is `pm.distributions.transform` no longer going to work?  It's in the `__all__` list in `transforms.py`, so I think it's still ok, isn't it?
But I like the idea of deprecating it, since for a class `Transform` is preferable.  Also `transform`, not being an initialized entity, doesn't serve the same purpose as the other lower-case names.